src/app/containers/PageHandlers/withData/index.jsx

```diff
@@ -1,21 +1,22 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { element, string } from 'prop-types';
 import pathOr from 'ramda/src/pathOr';
 import { dataPropType } from '../../../models/propTypes/data';
 import ErrorMain from '../../ErrorMain';
-import getPassportHome from '../../../lib/utilities/getPassportHome';
+import {
+  getPassportHome,
+  isValidPassportHome,
+} from '../../../lib/utilities/passport';
+import { ServiceContext } from '../../../contexts/ServiceContext';
 
 // checks for data, status, setting default status if not found
 const constructRenderObject = data => ({
   status: pathOr(null, ['status'], data) || 500,
   pageData: pathOr(null, ['pageData'], data),
 });
 
-const isValidPassportHome = (passportHome, service) =>
-  passportHome ? passportHome === service : true;
-
 // checks for pageData, 200 status and if home service from article data fits the service locale
-const shouldRender = (data, service) => {
+export const shouldRender = (data, service, passportHomesOverride = []) => {
```

Comment:
Why is this being exported?

Reply:
I've taken out `shouldRender` and its tests into their own folder

Reply:
Fixed!